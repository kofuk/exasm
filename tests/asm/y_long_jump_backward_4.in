@beg nop
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
j 2
lui r0, 1
nop
j @beg
nop
