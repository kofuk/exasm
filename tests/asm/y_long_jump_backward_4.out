@00 00000000 00000000 // nop
@02 00110000 00000001 // lui r0, 0x01
@04 11000000 00001010 // j 0x0A
@06 00110000 00000001 // lui r0, 0x01
@08 11000000 00000110 // j 0x06
@0a 00000000 00000000 // nop
@0c 11000000 11110010 // j -0x0E
@0e 00000000 00000000 // nop
@10 11000000 00000010 // j 0x02
@12 00110000 00000001 // lui r0, 0x01
@14 11000000 00000010 // j 0x02
@16 00110000 00000001 // lui r0, 0x01
@18 11000000 00000010 // j 0x02
@1a 00110000 00000001 // lui r0, 0x01
@1c 11000000 00000010 // j 0x02
@1e 00110000 00000001 // lui r0, 0x01
@20 11000000 00000010 // j 0x02
@22 00110000 00000001 // lui r0, 0x01
@24 11000000 00000010 // j 0x02
@26 00110000 00000001 // lui r0, 0x01
@28 11000000 00000010 // j 0x02
@2a 00110000 00000001 // lui r0, 0x01
@2c 11000000 00000010 // j 0x02
@2e 00110000 00000001 // lui r0, 0x01
@30 11000000 00000010 // j 0x02
@32 00110000 00000001 // lui r0, 0x01
@34 11000000 00000010 // j 0x02
@36 00110000 00000001 // lui r0, 0x01
@38 11000000 00000010 // j 0x02
@3a 00110000 00000001 // lui r0, 0x01
@3c 11000000 00000010 // j 0x02
@3e 00110000 00000001 // lui r0, 0x01
@40 11000000 00000010 // j 0x02
@42 00110000 00000001 // lui r0, 0x01
@44 11000000 00000010 // j 0x02
@46 00110000 00000001 // lui r0, 0x01
@48 11000000 00000010 // j 0x02
@4a 00110000 00000001 // lui r0, 0x01
@4c 11000000 00000010 // j 0x02
@4e 00110000 00000001 // lui r0, 0x01
@50 11000000 00000010 // j 0x02
@52 00110000 00000001 // lui r0, 0x01
@54 11000000 00000010 // j 0x02
@56 00110000 00000001 // lui r0, 0x01
@58 11000000 00000010 // j 0x02
@5a 00110000 00000001 // lui r0, 0x01
@5c 11000000 00000010 // j 0x02
@5e 00110000 00000001 // lui r0, 0x01
@60 11000000 00000010 // j 0x02
@62 00110000 00000001 // lui r0, 0x01
@64 11000000 00000010 // j 0x02
@66 00110000 00000001 // lui r0, 0x01
@68 11000000 00000010 // j 0x02
@6a 00110000 00000001 // lui r0, 0x01
@6c 11000000 00000010 // j 0x02
@6e 00110000 00000001 // lui r0, 0x01
@70 11000000 00000010 // j 0x02
@72 00110000 00000001 // lui r0, 0x01
@74 11000000 00000010 // j 0x02
@76 00110000 00000001 // lui r0, 0x01
@78 11000000 00000010 // j 0x02
@7a 00110000 00000001 // lui r0, 0x01
@7c 11000000 00000010 // j 0x02
@7e 00110000 00000001 // lui r0, 0x01
@80 11000000 00000010 // j 0x02
@82 00110000 00000001 // lui r0, 0x01
@84 11000000 00000010 // j 0x02
@86 00110000 00000001 // lui r0, 0x01
@88 00000000 00000000 // nop
@8a 11000000 10000000 // j -0x80
@8c 00000000 00000000 // nop
